dietpi/dietpi-software

```diff
@@ -8333,21 +8333,20 @@ _EOF_
 			[[ -d '/opt/filebrowser' ]] && G_EXEC rm -R /opt/filebrowser
 
 			# Install
-			G_EXEC cp -a filebrowser /opt/
-			G_EXEC rm -R filebrowser
+			G_EXEC mv filebrowser /opt/
 
 			# User
 			[[ -d '/mnt/dietpi_userdata/filebrowser' ]] || G_EXEC mkdir -p /mnt/dietpi_userdata/filebrowser
-			Create_User -G dietpi -d /mnt/dietpi_userdata/filebrowser filebrowser
+			Create_User -g dietpi -d /mnt/dietpi_userdata/filebrowser filebrowser
+			G_EXEC usermod -aG www-data filebrowser
```

Comment:
Ah hehe that was only an example of what users now can do to grant File Browser additional permissions. I think we don't want this by default 😉.

Reply:
so what should we install by default. I'm confused 🤣

Reply:
None required IMO. File Browser has "dietpi" as primary group and hence R/W access to `/mnt/dietpi_userdata` media and download directories, as well as exFAT drives (after this release) and network mounts, which should be good to start with. But I just want to keep the ability that users add File Browser to further groups, do grant it further permissions, if required. With `Group=dietpi` in the systemd service, this last aspect would not work (at least not reliably), so that I try to avoid it.